addon-test-support/-private/pick-last-locale.ts

```diff
@@ -11,4 +8,10 @@ import last from 'lodash.last';
  * @return {string}
  * @hide
  */
-export default locale => last(castArray(locale));
+export default function pickLastLocale(locale: string | string[]) {
+  if (Array.isArray(locale)) {
+    return locale[locale.length - 1];
+  }
+
+  return locale;
+}
```

Comment:
Some might say an arrow function would be easier to read:

```ts
export default (locale: string | string[]) =>
  Array.isArray(locale) ? locale[locale.length - 1] : locale;
```

But fine with me. 👍